.gitignore

```diff
@@ -1,6 +1,4 @@
-build
-build32
-build64
+build*
```

Comment:
i'd rather use:
`[bB]uild*`

Also on visual studio the default is an `out` folder.

Reply:
I'd suggest `/[bB]uild*/` with the first slash making it match only repository root and the final slash making it match folders only.

Reply:
Done.  I guess this can be merged into master now.